### ISSUE TITLE
RadioButton. Unselected options are not accessible

### ISSUE THREAD

**Issue by Kyzyl-ool** [gravity-ui/uikit#967]
Unselected options in [RadioButton](https://preview.gravity-ui.com/uikit/?path=/story/components-radiobutton--default) and [RadioGroup](https://preview.gravity-ui.com/uikit/?path=/story/components-radiogroup--default) components are not accessible from keyboard

**Comment by ValeraS** [comment]
What do you mean by "not accessible from keyboard"? When a group is focused, you can select options using the arrow keys.

**Comment by Kyzyl-ool** [comment]
> What do you mean by "not accessible from keyboard"? When a group is focused, you can select options using the arrow keys.

You are right, thank you. Did not expect that one can use arrow keys. Maybe one should provide an additional description about it?

**Comment by ValeraS** [comment]
> > What do you mean by "not accessible from keyboard"? When a group is focused, you can select options using the arrow keys.
> 
> You are right, thank you. Did not expect that one can use arrow keys. Maybe one should provide an additional description about it? At least accessible, via aria-describedby

It's standard browser behaviour, I think we don't need to describe it.

**Comment by amje** [comment]
Seems everything works fine. Closing this
